app/mailers/notification_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class NotificationMailer < ApplicationMailer
+  default from: 'notifications@example.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.